### PR TITLE
Remove margins from full screen layouts

### LINKED ISSUE #5415: Touch area vertically misaligned in full-screen
###### Reproduction Steps

1. set Settings -> Reviewing -> "Fullscreen mode" to "Hide the system bars and answer buttons" (also requires gestures to be set up)
2. have a card with buttons, i.e. "a href..." or via JavaScript
3. Review the card and try to press the button

###### Expected Result

Button is clickable/touchable

###### Actual Result

Button is not clickable, it doesn't open the link and/or performs the JavaScript after the first click. Some more clicks near and around the button are necessary in order to click it and perform its action.

With fullscreen mode "hide the system bars" and "none" there is no such problem. The buttons recognize every touch and work as expected.

###### Debug info
AnkiDroid Version = 2.9beta7

Android Version = 9

ACRA UUID = 1f29f610-893a-403f-83d5-25a26ba97fe0

###### Research
*Enter an [ x ] character to confirm the points below:*

[X  ] I have read the [support page](https://ankidroid.org/docs/help.html) and am reporting a bug or enhancement request specific to AnkiDroid

[X  ] I have checked the [manual](https://ankidroid.org/docs/manual.html) and the [FAQ](https://github.com/ankidroid/Anki-Android/wiki/FAQ) and could not find a solution to my issue

[ x ] I have searched for similar existing issues here and on the user forum

### PR DESCRIPTION
**Pull request by jgscherber**
## Purpose / Description
Removed margins from full screen. Originally added to allow pulling down and exit fullscreen, but the gesture of pulling down from the top appears to do that without the margin.

Also 

## Fixes
Fixes #5415

## Approach
Input area size matches display area size

## How Has This Been Tested?

Manually on API 25 and 29 VMs

## Learning (optional, can help others)
Used the Layout Inspector in Android Studio

## Checklist
_Please, go through these checks before submitting the PR._

- [x ] You have not changed whitespace unnecessarily (it makes diffs hard to read)
- [x ] You have a descriptive commit message with a short title (first line, max 50 chars).
- [x ] Your code follows the style of the project (e.g. never omit braces in `if` statements) 
- [ ] You have commented your code, particularly in hard-to-understand areas
- [ ] You have performed a self-review of your own code

### CODE REVIEW

**Comment by jgscherber** [comment]
Some questions are still open in #5415 but putting this up to start reviewing

**Comment by mikehardy** [comment]
Certainly laser-focused on the issue at hand, LGTM on a read but I don't have the wisdom in the area to approve or not quickly and I think Tim does so based on your git-blame-spelunking better to sit for the moment

**Comment by Anthropos888** [comment]
Any chance that this will be implemented soon? Since it fixes an annoying bug maybe the review high priority would be appropriate?

File: AnkiDroid/src/main/res/layout/reviewer_flashcard_fullscreen_noanswers.xml [hunk 1]
```diff
@@ -20,26 +20,22 @@
...
-    android:layout_margin="0dip">
+    android:layout_height="fill_parent">
 
     <FrameLayout
         android:layout_gravity="center"
         android:id="@+id/flashcard"
         android:layout_width="fill_parent"
-        android:layout_height="fill_parent" />
-    <!-- Whiteboard / gesture margin set on all sides so that immersive mode can be exited -->
+        android:layout_height="fill_parent" />S
```
Comment (mikehardy) [inline AnkiDroid/src/main/res/layout/reviewer_flashcard_fullscreen_noanswers.xml]: Looks like an unexpected `S` here?
```suggestion
        android:layout_height="fill_parent" />
```

**Comment by mikehardy** [comment]
Good point @Arthur-Milchior - I hadn't checked this in a while (sorry @jgscherber !) I saw one thing but otherwise I'd agree. Seems contained and ready to go minus the one item I mentioned in review

**Pull request merged**